### ISSUE TITLE
UDM-Pro support

### ISSUE THREAD

**Issue by platten** [Lochnair/vyatta-wireguard#138]
Hello,

Any chance a new build which would support the new Dream Machine Pro?

Thanks!

**Comment by TBK** [comment]
Would be relatively simple if the kernel code was available.

Try to start a forum thread on ui.com or shoot Ubnt support an email support@ui.com

**Comment by tomkv** [comment]
Correct me if I'm wrong, but UDM-Pro isn't running EdgeOS anywmore and doesn't allow configuring via config.gateway.json.

**Comment by TBK** [comment]
From what I have read is runs UniFi OS, what ever that is.

EdgeOS is UI's variant of vyatta which is a "Debian distro".

**Comment by tomkv** [comment]
Yes, and the previous routers - (or at least USG and USG-PRO-4) - are running EdgeOS, while UDM-PRO is not. UniFi OS is the new thing for UDM, which also means no config.gateway.json and no external controllers, only the built-in one.
